inference-engine/src/mkldnn_plugin/mkldnn_node.cpp

```diff
@@ -687,8 +690,14 @@ void MKLDNNNode::execute(mkldnn::stream strm) {
 
 void MKLDNNNode::executeDynamic(mkldnn::stream strm) {
     const auto newShapes = shapeInfer();
```

Comment:
**discuss** how to define that we don't need reallocate memory `bool shapeInfer(params)`
**discuss** need shape infer cache?